src/anomalib/data/errors.py

```diff
@@ -0,0 +1,13 @@
+class MisMatchError(Exception):
```

Comment:
Can you add docstring and licence header?

Reply:
"""Custom Exception Class for Mismatch Detection (MisMatchError)"""
# Copyright (C) 2025 Intel Corporation
# SPDX-License-Identifier: Apache-2.0

something like this?

Reply:
Yes! But it is still 2024 :)
You can have a look at how it is done in other files for example https://github.com/openvinotoolkit/anomalib/blob/main/src/anomalib/__init__.py#L1